Tests/XCUITests/EngagementNotificationTests.swift

```diff
@@ -0,0 +1,76 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+import XCTest
+
+class EngagementNotificationTests: BaseTestCase {
+    var currentScreen = 0
+    var rootA11yId: String {
+        return "\(AccessibilityIdentifiers.Onboarding.onboarding)\(currentScreen)"
+    }
+
+    override func setUp() {
+        launchArguments = [LaunchArguments.ClearProfile,
+                           "\(LaunchArguments.LoadExperiment)engagementNotificationWithoutConditions"]
+        super.setUp()
+    }
+
+    func testShowingNotification() {
+        goThroughOnboarding()
+
+        // As we cannot trigger the background refresh 
+        XCUIDevice.shared.press(XCUIDevice.Button.home)
+
+        let springboard = XCUIApplication(bundleIdentifier: "com.apple.springboard")
```

Comment:
Nit: you can remove this line, the springboard is already defined in the [FxScreenGraph](https://github.com/mozilla-mobile/firefox-ios/blob/de3eee1235768992eef87b9113916c47a386161e/Tests/XCUITests/FxScreenGraph.swift#L104)